o11y/honeycomb/honeycomb.go

```diff
@@ -65,7 +66,7 @@ func (c *Config) sender() transmission.Sender {
 				BatchTimeout:         libhoney.DefaultBatchTimeout,
 				MaxConcurrentBatches: libhoney.DefaultMaxConcurrentBatches,
 				PendingWorkCapacity:  libhoney.DefaultPendingWorkCapacity,
-				UserAgentAddition:    libhoney.UserAgentAddition,
+				UserAgentAddition:    c.ServiceName,
```

Comment:
The default user agent is `libhoney-go/<version>` this field adds the service name onto the end: `libhoney-go/<version> my-service`